### PR TITLE
[Lanaya]Cross-platform clipboard management with easy to use.

### PR DESCRIPTION
**Pull request by ChurchTao**
This is the link to the project: [Lanaya](https://github.com/ChurchTao/Lanaya)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [x] If the app closed source, evidence of it being built with Tauri is included.

### Additional Context

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [GitBar](https://github.com/mikaelkristiansson/gitbar) - System tray app for GitHub reviews.
 - [Wirefish](https://github.com/stefanodevenuto/wirefish) - Cross-platform packet sniffer and analyzer.
 - [Kanri](https://github.com/trobonox/kanri) - Cross-platform, offline-first Kanban board app with a focus on simplicity and user experience.
+- [Lanaya](https://github.com/ChurchTao/Lanaya) - Cross-platform clipboard management with easy to use.
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [Lanaya](https://github.com/ChurchTao/Lanaya) - Easy to use, cross-platform clipboard management.
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]
Please change the allowlist config to only allow the API features your app actually needs https://github.com/ChurchTao/Lanaya/blob/9927ceff37a4bba60414c8f7c670f84b256b523f/src-tauri/tauri.conf.json#L15
And consider adding a strict CSP https://github.com/ChurchTao/Lanaya/blob/9927ceff37a4bba60414c8f7c670f84b256b523f/src-tauri/tauri.conf.json#L55

**Comment by ChurchTao** [comment]
> Please change the allowlist config to only allow the API features your app actually needs https://github.com/ChurchTao/Lanaya/blob/9927ceff37a4bba60414c8f7c670f84b256b523f/src-tauri/tauri.conf.json#L15 And consider adding a strict CSP https://github.com/ChurchTao/Lanaya/blob/9927ceff37a4bba60414c8f7c670f84b256b523f/src-tauri/tauri.conf.json#L55

Thanks for the reminder, it has been corrected. 😄

**Comment by JonasKruckenberg** [comment]
> script-src 'unsafe-eval' 'self'; default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self'; img-src data: 'self';

This CSP is the exact opposite of strict 😄 `default-src` with that set of arguments essentially means you still allow almost all connections and kinds of resources.

**Comment by JonasKruckenberg** [comment]
Also, before I forget again: We require all commits to be signed across Tauri's repositories. Follow [this guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) to set up commit signing.

**Comment by ChurchTao** [comment]
I resubmitted a signed version in [https://github.com/tauri-apps/awesome-tauri/pull/116](url)

**Pull request closed without merging**